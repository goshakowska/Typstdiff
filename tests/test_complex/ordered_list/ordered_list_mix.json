{"pandoc-api-version":[1,23,1],"meta":{},"blocks":[{"t":"OrderedList","c":[[1,{"t":"DefaultStyle"},{"t":"DefaultDelim"}],[[{"t":"Para","c":[{"t":"Str","c":"The"},{"t":"Space"},{"t":"Str","c":"natural"},{"t":"Space"},{"t":"Str","c":"climate"}]},{"t":"BulletList","c":[[{"t":"Para","c":[{"t":"Str","c":"Temperature"},{"t":"Space"},{"t":"Str","c":"factors"}]},{"t":"OrderedList","c":[[1,{"t":"DefaultStyle"},{"t":"DefaultDelim"}],[[{"t":"Para","c":[{"t":"Str","c":"degree"}]},{"t":"BulletList","c":[[{"t":"Para","c":[{"t":"Str","c":"hot"}]}],[{"t":"Para","c":[{"t":"Str","c":"freezing"}]}]]}],[{"t":"Para","c":[{"t":"Str","c":"sun"}]}],[{"t":"Para","c":[{"t":"Str","c":"wind"}]}]]]}]]}],[{"t":"Para","c":[{"t":"Str","c":"The"},{"t":"Space"},{"t":"Str","c":"geology"}]}],[{"t":"Para","c":[{"t":"Str","c":"Something"},{"t":"Space"},{"t":"Str","c":"new"}]}],[{"t":"Para","c":[{"t":"Str","c":"Monkey"}]}]]]}]}
