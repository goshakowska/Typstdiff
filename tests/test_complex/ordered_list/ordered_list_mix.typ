+ The climate
  - Precipitation    
  - Temperature factors
    + degree 
        - hot
        - cold
+ Something new
+ Monkey