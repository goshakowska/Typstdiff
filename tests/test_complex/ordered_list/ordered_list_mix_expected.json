{"pandoc-api-version": [1, 23, 1], "meta": {}, "blocks": [{"t": "OrderedList", "c": [[1, {"t": "DefaultStyle"}, {"t": "DefaultDelim"}], [[{"t": "Para", "c": [{"t": "Str", "c": "The"}, {"t": "Space"}, {"t": "Str", "c": "climate"}]}, {"t": "BulletList", "c": [[{"t": "Para", "c": [{"t": "Str", "c": "Precipitation"}]}], [{"t": "Para", "c": [{"t": "Str", "c": "Temperature"}, {"t": "Underline", "c": [{"t": "Space"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "factors"}]}]}, {"t": "OrderedList", "c": [[1, {"t": "DefaultStyle"}, {"t": "DefaultDelim"}], [[{"t": "Para", "c": [{"t": "Str", "c": "degree"}]}, {"t": "BulletList", "c": [[{"t": "Para", "c": [{"t": "Str", "c": "hot"}]}], [{"t": "Para", "c": [{"t": "Str", "c": "cold"}]}], [{"t": "Para", "c": [{"t": "Strikeout", "c": [{"t": "Str", "c": "warm"}]}]}]]}], [{"t": "Para", "c": [{"t": "Strikeout", "c": [{"t": "Str", "c": "sun"}]}]}]]]}]]}], [{"t": "Para", "c": [{"t": "Strikeout", "c": [{"t": "Str", "c": "The"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "Something"}]}, {"t": "Space"}, {"t": "Strikeout", "c": [{"t": "Str", "c": "geology"}]}, {"t": "Underline", "c": [{"t": "Str", "c": "new"}]}]}], [{"t": "Para", "c": [{"t": "Underline", "c": [{"t": "Str", "c": "Monkey"}]}]}]]]}]}